releasenotes/notes/0.4/experiment_data_refactor-1bb5ba366fb09bc5.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
```

Comment:
If this indicates user must install new package, this should be `upgrade` notice.